features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -76,6 +78,16 @@ data class SessionReplayConfiguration internal constructor(
             return this
         }
 
+        /**
+         * Sets the dynamic optimisation for the Session Replay feature.
+         * This will enable the time limitation that Session Replay recording can use on the main thread, enabling it
+         * might reduce the session replay frames recorded by the SDK.
```

Comment:
```suggestion
         * This will enable the time limitation that Session Replay recording can use on the main thread. Enabling it
         * might reduce the session replay frames recorded by the SDK.
```

but still, for the customers this explanation is somewhat cryptic. Maybe we should re-word it a bit simpler/more understandable? Something like:

```
This option enables/disables the optimization, which can limit amount of the Session Replay data recorded. If optimization is enabled, there is a time quota per second which Session Replay can utilize to record the data.
```

A bit simplified, but you see the idea.

Reply:
Idk if we should specifically refer to specfic mechanisms in the comment for this parameter, because it might equally apply for disabling the sdk on weak devices on startup, or any other type of dynamic performance improvement that we want to do now or in the future.

Reply:
I reverse the meaning of the parameter, now it changed from `dynamicOptimization` to `disableDynamicOptimization`, document updated as well, please check it again